mongo_orchestration/apps/servers.py

```diff
@@ -30,7 +30,7 @@
 from mongo_orchestration.errors import RequestError
 from mongo_orchestration.servers import Servers
 
-logging.basicConfig(level=logging.DEBUG)
```

Comment:
The default level is WARNING, which does not include INFO or DEBUG.

https://docs.python.org/2/howto/logging.html#when-to-use-logging